### ISSUE TITLE
--project does not exclude file

### ISSUE THREAD

**Issue by ghost** [palantir/tslint#3913]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.3.4
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
// (not applicable)
```

with `tslint.json` configuration:

```json
{
	"exclude": [
		"myexcludedfolder/**/*.ts"
	]
}
```

#### Actual behavior
When running tslint with "--project tsconfig.json" with a tslint.json configuration that excludes the folder "myexcludedfolder", tslint still lints "myexcludedfolder".

#### Expected behavior
I was expecting that tslint wouldn't lint the folder "myexcludedfolder".

Perhaps I am misunderstanding the purpose of "--project"?

**Comment by kylecordes** [comment]
It appears the only way to have "exclude" exclude, is to structure things so that you do not need to use `--project`?  Yet without that setting, some rules are unworkable (requiring type information).  The workaround for now appears to do the exclusion the old way, with a long series of command line arguments.

**Comment by giladgray** [comment]
@kylecordes can you exclude these files from the tsconfig project instead? if not, then i think you're stuck linting them.

**Comment by kylecordes** [comment]
@giladgray Yes, I was able to get what I need working... but a user should still expect both the project and exclude settings to work "as advertised".

**Comment by JoshuaKGoldberg** [comment]
Something like a duplicate of #3623.
